src/mom5/ocean_param/gotm-4.0/turbulence/turbulence.F90

```diff
@@ -416,7 +417,9 @@ subroutine init_turbulence(namlst,fn,nlev)
 !
 !-----------------------------------------------------------------------
 !BOC
+   if(mpp_pe() == mpp_root_pe()) then
    LEVEL1 'init_turbulence: v',RELEASE
+   endif
```

Comment:
Note for future patch, should replace these LEVEL# macros with `stdoutunit = stdout()` and `write(stdoutunit, *) ...` calls

Reply:
Or even just replace `STDERR` with
```
stderrunit = stderr()
#define STDERR write(stderrunit, *)
```
to eliminate these `mpp_pe` checks.